stremio-core-protobuf/src/bridge/resource_loadable.rs

```diff
@@ -1,18 +1,43 @@
 use inflector::Inflector;
+use url::Url;
+
 use stremio_core::deep_links::DiscoverDeepLinks;
 use stremio_core::models::common::{DescriptorLoadable, ResourceLoadable};
 use stremio_core::models::ctx::Ctx;
 use stremio_core::types::addon::{DescriptorPreview, ResourceRequest};
 use stremio_core::types::library::LibraryItem;
 use stremio_core::types::resource::{MetaItem, MetaItemPreview, Stream, Subtitles};
-use stremio_watched_bitfield::WatchedBitField;
-use url::Url;
+use stremio_core::types::watched_bitfield::WatchedBitField;
 
 use crate::bridge::ToProtobuf;
 use crate::protobuf::stremio::core::models;
+// use crate::stremio_core_models::types::ResourcePath;
+
+// impl<'a> ToProtobuf<models::LoadablePage, Ctx> for stremio_core_web::model::serialize_catalogs_with_extra::ResourceLoadable<'a> {
```

Comment:
Commened code?

Reply:
Yes, I tried to implement conversion between the 2 structs (core-web and core-protobuf) but fields and other stuff differ so I left it here for implementation. Will remove it